response_play.go

```diff
@@ -0,0 +1,111 @@
+package pagerduty
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+type ResponsePlay struct {
+	APIObject
```

Comment:
Could you please remove this `APIObject` embed, and instead manually enumerate all the `APIObject` fields directly on this struct type? We're looking to remove this pattern in v2+, and so may as well try to avoid adding more cases of it. There's a little more context in #318 if you're curious.